runtime/src/bank.rs

```diff
@@ -477,6 +477,42 @@ pub(crate) struct BankFieldsToSerialize<'a> {
     pub(crate) is_delta: bool,
 }
 
+impl PartialEq for Bank {
```

Comment:
By the way, why can't we just `derive` `PartialEq`? A comment might be desirable to explain the limitation.

Reply:
hehe so the issue was RwLock doesn't implement PartialEq, and we can't implement it for RwLock because it doesn't belong to this crate.